docs/content/guides/technical-specification/mobile-support.md

```diff
@@ -0,0 +1,113 @@
+| Scroll vertically                    | Swipe up or down                                 | &check; | &check; |
+| Scroll horizontally                  | Swipe left or right                              | &check; | &check; |
+| Change to portrait or landscape mode | Rotate the screen                                | &check; | &check; |
+| Open the column menu                 | In a column header, tap &#9660;                  | &check; | &check; |
+| Open the context menu                | Long-press a cell                                | &check; | &cross; |
+| Interact with menus                  | Tap a menu item                                  | &check; | &check; |
+
+#### Selection handles
+
+To select a range of cells, drag the selection handles that display only on mobile:
+
+![Selection handles]({{$basePath}}/img/selection_handles.png)
+
+## Features supported on mobile
+
+As desktop use cases are our priority, we test only selected Handsontable functionalities on mobile.
+See the following table for the list of features that work
+on the [supported mobile browsers](@/guides/technical-specification/supported-browsers.md#supported-mobile-browsers):
+
+- &check; We regularly test these features on a limited number of physical devices.
+- &cross; These features may work properly on mobile, but we don't test them on mobile devices.
+
+| Feature                                                                    | Android (Chrome) | iOS (Chrome) | iOS (Safari) |
+| -------------------------------------------------------------------------- | :--------------: | :----------: | :----------: |
+| All the built-in [cell types](@/guides/cell-types/cell-type.md)            |     &check;      |   &check;    |   &check;    |
+| Insert rows                                                                |     &check;      |   &check;    |   &check;    |
+| Insert columns                                                             |     &check;      |   &check;    |   &check;    |
+| Remove rows                                                                |     &check;      |   &check;    |   &check;    |
+| Remove columns                                                             |     &check;      |   &check;    |   &check;    |
+| [Column headers](@/guides/columns/column-header.md)                        |     &check;      |   &check;    |   &check;    |
+| [Column groups](@/guides/columns/column-groups.md)                         |     &check;      |   &check;    |   &check;    |
+| [Column hiding](@/guides/columns/column-hiding.md)                         |     &check;      |   &check;    |   &check;    |
+| [Column moving](@/guides/columns/column-moving.md)                         |     &cross;      |   &cross;    |   &cross;    |
+| [Column freezing](@/guides/columns/column-moving.md)                       |     &cross;      |   &cross;    |   &cross;    |
```

Comment:
Wrong link. It can be marked as "checked" . I test it on Android using the context menu and during testing documentation on iOS.